DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
+Section \ref{sect:detailedtestresults} provides detailed results for each step in each test case.
 
-Section \ref{sect:testresults} is filled after the test activity is completed. 
-Its includes  an overview of the resulta in \ref{sect:overview}
-while \ref{sect:detailedtestresults} provides more detailed results from each individual test case.
+The current status of test plan LVV-P10 in Jira is Draft.
 
+\subsection{References}
+\label{sect:references}
+\renewcommand{\refname}{}
+\bibliography{lsst,refs,books,refs_ads}
 \section{Test Configuration}
 \label{sect:configuration}
 
-The configuration for this test is the data is located in a file system somewhere.  It is transferred to NCSA into a the LSST GPFS file system.   Files that are needed in the DBB are moved to the DBB ingest file system.   the data backbone gathers the info from the headers and file name.  it inserts rows into the DBB consolidated DB environment and moves the file to the DBB file systems that are used by the LSP to once again find the raw image file to be viewed/displayed for further analysis.  
-
-\subsection{Hardware}
-\label{sect:hwconf}
+\subsection{Data Collection}
 
-LDF center at NCSA.   The archive service or data back bone.
+  Observing is not required for this test campaign.
 
-\subsection{Software}
-\label{sect:swconf}
+\subsection{Verification Environment}
+\label{sect:hwconf}
+  LDF center at NCSA. ~ The archive service or data back bone.~~
 
-The DBB version is 1.0, and can be found in the github repository when it has been fully developed.   The current DBB is a script for scraping data from the header file and getting the file name and the metadata required along with providence.   At the time of this writing, the DBB use cases and requirements are just being written.    The database component is Oracle and the ingest file systems and output file systems for DBB are in GPFS at the LDF.  
 
-\subsection{Entry Criteria}
+  \subsection{Entry Criteria}
+  Data that needs to be stored from a CCOB or DAQ
 
-Data that needs to be stored from a CCOB or DAQ.
 
-\subsection{Exit Criteria}
+  \subsection{Exit Criteria}
+  Data is transferred in proper form and can be viewed on LSP after being
+transferred to LDF and ingested into DBB.~~
 
-Data is transferred in proper form and can be viewed on LSP after being transferred to LDF and ingested into DBB.  
 
 
 \section{Personnel}
 \label{sect:personnel}
 
-Following personnel is involved in the test activity:
+Following personnel are involved in this test activity:
 
 \begin{itemize}
-\item Test Plan (LVV-P10) Owner: Michelle Butler
-\item Test Cycle (LVV-C8) Owner: Unassigned
-\item LVV-T284 Tester: Michelle Butler
-\item Additional Test Personel involved: None 
+\item Test Plan (LVV-P10) owner: Michelle Butler
+\item Test Cycles:
+\begin{itemize}
+  \item LVV-C8 owner: 
+    Undefined
```

Comment:
Should have an owner.

Reply:
The tester is specified for each test case.

Reply:
I put a tester on each test that was human done.